src/main/scala/com/metl/data/MessageBus.scala

```diff
@@ -20,7 +24,7 @@ abstract class MessageBusProvider {
   def sendMessageToBus(busFilter:MessageBusDefinition => Boolean, message:MeTLStanza):Unit = {}
 }
 abstract class OneBusPerRoomMessageBusProvider extends MessageBusProvider {
-  protected lazy val busses = new SynchronizedWriteMap[MessageBusDefinition,MessageBus](scala.collection.mutable.HashMap.empty[MessageBusDefinition,MessageBus],true,(k:MessageBusDefinition) => createNewMessageBus(k))
+  protected lazy val busses = JavaConversions.mapAsScalaMap(new ConcurrentHashMap[MessageBusDefinition,MessageBus]())
```

Comment:
I think you'll want to use JavaConversions.mapAsScalaConcurrentMap instead of mapAsScalaMap